index.js

```diff
@@ -0,0 +1,97 @@
+import { catsData } from "/data.js"
+
+const catsArray = []
+const emotionsArray = []
+const emotionRadios = document.getElementById('emotion-radios')
+const radios = document.getElementsByClassName('radio')
+const getImageBtn = document.getElementById('get-image-btn')
+const gifsOnlyCheckbox = document.getElementById('gifs-only-option')
+const memeModalInner = document.getElementById('meme-modal-inner')
+const memeModal = document.getElementById('meme-modal')
+const memeModalClose = document.getElementById('meme-modal-close-btn')
+
+emotionRadios.addEventListener('change', highlightCheckedOption)
+getImageBtn.addEventListener("click", renderCat)
+memeModalClose.addEventListener("click", closeModal)
+
+function closeModal() {
+    memeModal.style.display = "none"
+}
+
+function getMatchingCatsArray() {
```

Comment:
```suggestion
function getMatchingCats() {
```

Reply:
Addressed, updated function name to exclude type